python/ray/serve/tests/conftest.py

```diff
@@ -151,6 +151,28 @@ def ray_start_stop():
     )
 
 
+@pytest.fixture(scope="function")
+def ray_start_stop_in_specific_directory(request):
+    # Change working directory so Ray will start in the correct directory.
+    working_dir = request.param
+    os.chdir(working_dir)
```

Comment:
Could you print the directory that this test runs in? If something changes in our CI, which somehow breaks the test, that could help speed up debugging.

Could you also print the directory after you change it back at the end of this fixture?